tests/backstop_data/backstop.tpl.json

```diff
@@ -22,7 +22,7 @@
       "--no-sandbox"
     ]
   },
-  "asyncCaptureLimit": 3,
+  "asyncCaptureLimit": 1,
```

Comment:
why was this reduced?

Reply:
More than 1 Using more than 1 thread creates lot of false negatives. But with 1, its much more reliable.